packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -102,10 +136,163 @@ class CupertinoContextMenu extends StatefulWidget {
   CupertinoContextMenu({
     super.key,
     required this.actions,
-    required this.child,
-    this.previewBuilder,
+    this.child,
+    this.builder,
+    @Deprecated(
+      'Use CupertinoContextMenu.builder instead. '
+      'This feature was deprecated after v3.4.0-34.1.pre.',
+    )
+    this.previewBuilder = _defaultPreviewBuilder,
   }) : assert(actions != null && actions.isNotEmpty),
-       assert(child != null);
+       assert(child != null || builder != null,
+        'Either a child or a builder is required'
+       ),
+       _builder = builder ?? ((BuildContext context, Animation<double> animation) => child!);
+
+  /// Exposes the default border radius for matching iOS 16.0 behavior. This
+  /// value was eyeballed from the iOS simulator running iOS 16.0.
+  ///
+  /// {@tool snippet}
+  ///
+  /// Below is example code in order to match the default border radius for an
+  /// iOS 16.0 open preview.
```

Comment:
Actually this example shows the border start to animate even before the "open" state, which is not correct for iOS 16, right?

Also it lacks the drop shadow.  I guess that might be irrelevant for kOpenBorderRadius, but maybe you should mention that, or point to the full example below as well.